CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Badge to README to show code coverage percentage.
 - Support for Python 3.12.
+- Additional data classes that draw distinction between supervised and
```

Comment:
Use fully qualified names here. E.G. `coreax.data.Data` and `coreax.data.SupervisedData` and mention that these provide handling of weighted data.

Reply:
Agree, done